apps/omg_watcher/lib/web/controllers/transaction.ex

```diff
@@ -52,4 +53,68 @@ defmodule OMG.Watcher.Web.Controller.Transaction do
 
   # Responds when the transaction is not found
   defp respond_single(nil, conn), do: send_resp(conn, :not_found, "")
+
+  def swagger_definitions do
+    %{
+      Transaction:
+        swagger_schema do
+          title("Transaction")
+
+          properties do
+            txid(:string, "Transaction id", required: true)
+            blknum1(:integer, "Childchain block number of first input utxo", required: true)
```

Comment:
Pls add `the` before each `first`, `second`, e.g. "Childchain block number of the first input utxo"

Reply:
done